roadmap/1_core-protocol/in-progress/12_composite_queries.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Composite queries
+links:
+  Forum Link: https://forum.dfinity.org/t/proposal-composite-queries/15979
+  Proposal: https://dashboard.internetcomputer.org/proposal/87599
+eta: 2022
```

Comment:
Are we sure this is feasible? I want to avoid overpromising

Reply:
Good point. I bumped it up to 2023. We can adjust it when we are closer to the release.

Reply:
@dprats: could you please take another look?